charts/gateway-helm/README.md

```diff
@@ -55,6 +59,16 @@ To uninstall the chart:
     helm delete eg
 ```
 
+### Certgen
+
+Certgen is used to generate the certificates required by EnvoyGateway.
```

Comment:
It also creates the HMAC secret for verifying OAuth2 cookies.

Reply:
Thank you for your supplement!